app/src/main/res/values-it/strings.xml

```diff
@@ -21,148 +21,148 @@
-    <string name="inform_title_homescreen">"Verdacht auf eine Corona Erkrankung"</string>
-    <string name="inform_text_homescreen">"Informiere Personen, denen du begegnet bist, wenn du Symptome verspürst oder positiv getestet wurdest."</string>
-    <string name="inform_button_title">"Erkrankung melden"</string>
-    <string name="inform_button_title_again">"Erneut informieren"</string>
-    <string name="inform_last_meldung_text">"Letzte erfolgreiche Meldung am {date}"</string>
+    <string name="inform_title_homescreen">"Sospetto COVID-19"</string>
+    <string name="inform_text_homescreen">"Informa le persone che hai recentemente incontrato che hai i sintomi o il test al COVID-19 è risultato positivo."</string>
+    <string name="inform_button_title">"Segnala adesso"</string>
+    <string name="inform_button_title_again">"Aggiorna la segnalazione"</string>
+    <string name="inform_last_meldung_text">"Ultima segnalazione effettuata {date}"</string>
 
     <!-- Erstes Tab -->
-    <string name="tab_theapp_title">"Die App"</string>
+    <string name="tab_theapp_title">"L'App"</string>
 
     <!-- Tab Titel -->
-    <string name="tab_aktuell_title">"Aktuell"</string>
+    <string name="tab_aktuell_title">"Adesso"</string>
 
     <!-- Tab Titel -->
-    <string name="tab_verhalten_title">"Verhalten"</string>
+    <string name="tab_verhalten_title">"Comportamento"</string>
 
     <!-- Tab Titel -->
-    <string name="tab_prinzip_title">"Prinzip"</string>
+    <string name="tab_prinzip_title">"Principio"</string>
 
     <!-- Abbrechen -->
-    <string name="cancel">"Abbrechen"</string>
+    <string name="cancel">"Annulla"</string>
 
     <!-- Schliessen -->
-    <string name="close">"Schliessen"</string>
+    <string name="close">"Chiudi"</string>
 
     <!-- App Name ohne COVID-19 -->
     <string name="app_name_star">"Next Step"</string>
 
     <!-- Info Text ohne COVID-19 -->
-    <string name="homescreen_handshakes_module_info_label_star">"Zur Zeit hat kein Nutzer, der kürzlich in deiner Nähe war, ein Signal aktiviert."</string>
+    <string name="homescreen_handshakes_module_info_label_star">"Non ci sono segnalazioni di contatti con persone infette."</string>
 
     <!-- Text bei aktivierung ohne COVID-19 -->
-    <string name="homescreen_handshakes_module_gemeldet_label_star">"Ein Nutzer, der kürzlich in deiner Nähe war, hat ein Signal aktiviert."</string>
+    <string name="homescreen_handshakes_module_gemeldet_label_star">"Una persona con cui sei stato recentemente in conttato ha inviato una segnalazione."</string>
 
     <!-- Informieren Titel ohne COVID-19 -->
     <string name="inform_title_homescreen_star">"Broadcast"</string>
 
     <!-- Informieren Info Text ohne COVID-19 -->
-    <string name="inform_text_homescreen_star">"Sende ein Signal an Nutzer, denen du kürzlich begegnet bist."</string>
+    <string name="inform_text_homescreen_star">"Manda una segnalazione a tutte le persone che sono state in contatto con te."</string>
```

Comment:
I would prefer to use "Avverti tutte le persone che sono state in contatto con te.". I don't really like the word "Segnalazione". What do you thing?

Reply:
Si tutto gli usi di "segnalazione" non mi tornano. Per adesso li lascereri così, poi vediamo se sostituirli tutti con "Avvertimento".

Reply:
Ad essere onesto nemmeno "Avvertimento" secondo me suona bene, magari se vuoi restare con "Segnalazione", si potrebbero cambiare tutti gli "Manda una segnalazione" con "Segnala bla bla". To be honest also "Avvertimento" doesn't sounds good to me, if you want to keep "Segnalazione", we could change al the "Manda una segnaglazione" with just "Segnala bla bla"

Reply:
Avvertimento mi sembra un po' troppo siciliano, compari.

Magari notifica? Sennó segnalazione mi sembra che per ora vada bene